util/dogfood.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 rm -rf target*/*so
-cargo build --lib && cp -R target target_recur && cargo rustc -- -Zextra-plugins=clippy -Ltarget_recur/debug -Dclippy_pedantic -Dclippy || exit 1
+cargo build --lib && cp -R target target_recur && cargo rustc --lib -- -Zextra-plugins=clippy -Ltarget_recur/debug -Dclippy_pedantic -Dclippy || exit 1
```

Comment:
Cargo test will actually run a separate implementation of dogfood, so there wasn't any need for that script as far as I knew